src/components/Checkout/LoanMatcher.vue

```diff
@@ -1,7 +1,7 @@
 <template functional>
 	<p class="tw-text-small tw-text-secondary matching-text">
 		<!-- eslint-disable-next-line max-len -->
-		<span :class="props.disableMatching && 'tw-line-through'">Matched by {{ props.matchingText }}</span>{{ props.disableMatching ? '. Loan matching only applies when you pay in full with new funds using Card, Paypal, Google Pay, or Apple Pay.' : '' }}
+		<span class="tw-line-through">Matched by {{ props.matchingText }}</span>
```

Comment:
Do we want this `tw-line-through` class here? It looks like it was only added before when `props.disableMatching` was `true`.

Reply:
even the span, thanks for your very sharp-eyed way!